### DISCUSSION TITLE
[Question][KCLOUD] KCLOUD VM and environment

### DISCUSSION THREAD

**Discussion by miraliahmadli** [prosyslab-classroom/cs348-information-security#612] [Q&A]
I cannot access neither my KCLOUD account nor my VM. I emailed to KCLOUD staff on this issue, but they didn't reply and I cannot access to my account since then. Please allocate new VM and account for me or fix my issue or allocate any other cloud service.

I did not choose to use KCLOUD for the assignments and this assignments are heavily environment dependent. I cannot setup proper environment on my Mac because we are using LLVM 10 (and that is the only version not available on Homebrew). Moreover, if I have any issue with KCLOUD, I should not contact to them, I should contact to the people who chose that environment and they have to fix the issue.

Also, docker is very problematic as I cannot open it in my code editor and I am forced to work using VIM (which I am not comfortable at all) as well as I have to make changes locally then push to GitHub then pull it on docker. Therefore, running and testing code takes a lot of time.

I used docker on last 2 assignments, but it was very painful. Fortunately, assignments were not that hard and didn't require much debugging. But I cannot do the same with this assignment.

PLEASE, FIX MY KCLOUD ISSUE OR ALLOCATE NEW ACCOUNT.

**Comment by jirheee** [comment]
You can use vscode docker extension to access the  container. try this!

**Comment by nv-quan** [comment]
Did you try editing on your host machine and run on the container?

**Comment by RiceBiscuits** [comment]
I'm sorry to hear that.
Unfortunately, there are nothing we can do regarding KCloud issues without support from the KCloud administrator. 
So, I think you should wait until the problem is solved.
 
If you have a problem using  KCloud and docker, you can set up your own environment with a virtual machine program(i.e., VMware Fusion, Virtual Box, Parallels).
KCloud and docker are just options.

**Comment by RiceBiscuits** [comment]
Please check the vm and KCloudVPN again, if the problem is not solved yet, please send an email to kcloud@kaist.ac.kr again , with your IP address of the vm.
